src/messages/mod.rs

```diff
@@ -331,8 +331,11 @@ impl SignedRoutingMessage {
                 } else {
                     log_or_panic!(
                         LogLevel::Error,
-                        "Combining signatures failed on {:?}!",
-                        self
+                        "Combining signatures failed on {:?}! Part Shares: {:?}, Part Set: {:?}, Partial: {:?}",
+                        self,
+                        partial.shares,
+                        partial.pk_set,
+                        partial,
```

Comment:
Why do we need to output the shares and the pk_set separately when we also output the whole `partial` thing anyway?

Reply:
Because the whole partial omit these as irrelevant (which is true in most cases, but not here).